### ISSUE TITLE
Pip installable

### ISSUE THREAD

**Issue by ArianeMora** [ComputationalSystemsBiology/OT-scOmics#1]
Hi! We've been using your package as part of a benchmarking pipeline, however, when packaging our pipeline it is non-trivial to include a non-pip installable package like otscomics (currently we have a not so nice workaround so would be great to be able to just `pip install otscomics`).
The process is: 1) sign up to pip: https://pypi.org/account/register/, 2) in your OT-scOmics directory run: `python setup.py sdist bdist_wheel`, 3) check your built package (e.g. create a fresh env and try running your notebooks), 4) use twine (install via https://pypi.org/project/twine/) to check the package: `twine check dist/*` finally, 5) upload to pip: `twine upload dist/*`. Perhaps a thought would be to provide two builds (with and without cuda enabled) to make it accessible to different envs. Let me know if you have issues and please let me know when it becomes available, cheers!

**Comment by gjhuizing** [comment]
Hi! Glad to know you're interested in our tool :) No problem, give me a few days and I'll set it up! In the meantime, have you tried `pip install git+https://github.com/ComputationalSystemsBiology/OT-scOmics.git` ?

**Comment by ArianeMora** [comment]
Yes! We are able to install and run `otscomics` :) the problem arises when trying to distribute our benchmark via pip (pip doesn't let you reference non-pip installable packages in the setup.py file which is where our issue with `otscomics` arose). At the moment we just have documentation for people to separately install `otscomics` but it would be nice to be able to include it by default :)

**Comment by gjhuizing** [comment]
Hi again!

- :partying_face: The package is now pip installable
- :warning: I used the opportunity to update the code. We now depend on [POT](https://pythonot.github.io/) to do the computations. The API thus has some minor changes, you might need to update your code.
- :book: Finally, I made a short documentation page and moved the main vignette there. [Documentation](https://ot-scomics.readthedocs.io/en/latest/)

I hope this helps, and let me know if the changes broke anything!
